package.json

```
@@ -19,7 +19,9 @@
     "verify": "npm run lint && npm run test",
     "format": "prettier --write \"src/**/*.ts\" \"src/**/*.js\"",
     "storybook": "BROWSER=NONE start-storybook -p 6006",
-    "build-storybook": "build-storybook"
+    "build-storybook": "build-storybook",
+    "docs": "start-storybook --docs",
```

```suggestion
    "docs": "npm run storybook -- --docs",
```
unless you want to override the flags
also this adds composability

---

I see your point, ok, I'll do it as u suggest also for the build-docs (although it has no args)